tests/unit_tests/test_wave_03.py

```diff
@@ -130,8 +130,9 @@ def test_swap_items_from_their_empty_returns_false():
     nobodys_item = Item()
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
+    result == False 
```

Comment:
Similar feedback as on the previous test, what other aspects could we confirm to ensure there were no unintended changes?